wdl/src/main/scala/wdl/WdlNamespace.scala

```diff
@@ -480,16 +481,14 @@ object WdlNamespace {
       }
     }
 
-    /*
-     * Ensures that the lhs corresponds to a call and the rhs corresponds to one of its outputs. We're only checking
-     * top level MemberAccess ASTs because the sub-ASTs don't make sense w/o the context of the parent. For example
-     * if we have "input: var=ns.ns1.my_task" it does not make sense to validate "ns1.my_task" by itself as it only
-     * makes sense to validate that "ns.ns1.my_task" as a whole is coherent
-     *
-     * Run for its side effect (i.e. Exception) but we were previously using a Try and immediately calling .get on it
-     * so it's the same thing
-     */
-    val invalidMemberAccesses = callInputSections flatMap { ast =>
+//    /*
+//     * Ensures that the lhs corresponds to a call and the rhs corresponds to one of its outputs. We're only checking
+//     * top level MemberAccess ASTs because the sub-ASTs don't make sense w/o the context of the parent. For example
+//     * if we have "input: var=ns.ns1.my_task" it does not make sense to validate "ns1.my_task" by itself as it only
+//     * makes sense to validate that "ns.ns1.my_task" as a whole is coherent
+//     *
+//     */
```

Comment:
um what

Reply:
😳 this probably got swept up in a big <kbd>ALT</kbd> <kbd>/</kbd> and then didn't get unswept by the subsequent "reinstate everything"